tidy3d/components/simulation.py

```diff
@@ -1173,8 +1173,26 @@ def plot_grid(  # pylint:disable=too-many-locals
         segs_y = [((xmin, bound), (xmax, bound)) for bound in boundaries_y]
         line_segments_y = mpl.collections.LineCollection(segs_y, **kwargs)
 
+        # Plot grid
         ax.add_collection(line_segments_x)
         ax.add_collection(line_segments_y)
+
+        # Plto bounding boxes of override structures
+        patch_kwargs = {
```

Comment:
the way the plotting is set up now it might be good to define these here
https://github.com/flexcompute/tidy3d/blob/develop/tidy3d/components/viz.py#L108
import them in this file, and then call
`plot_params_override_structures.to_kwargs()`
and pass `zorder` in `Rectangle`

Reply:
viz.py
```python
# add `fill : bool = True` to `PlotParams`
plot_params_override_structures = PlotParams(linewidth=.4, edgecolor='black', fill=False)
```

simulation.py
```
from .viz import plot_params_override_structures
# ...
plot_params = plot_params_override_structure.copy(deep=True)
plot_params.include_kwargs(lw=2 * kwargs["linewidth"], ec=kwargs["colors"])
patch_kwargs = plot_params.to_kwargs()
```